docs/conf.py

```diff
@@ -0,0 +1,63 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=invalid-name
+
+"""
+Sphinx documentation builder
+"""
+
+# General options:
+
+project = "Template project"
+copyright = "2022"  # pylint: disable=redefined-builtin
+author = ""
+
+# The short X.Y version
+version = ""
+# The full version, including alpha/beta/rc tags
+release = "0.0.0"
```

Comment:
We might need to find a way to keep these up to date, perhaps using a `VERSION.txt` file like @calebj15 has been talking about.

Reply:
probably versions file is a good idea. I will push that in a following commit.